From f63b6427d7c14b0ec92082b0ec9c77bafa17877c Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Sun, 7 Aug 2016 23:07:32 -0700
Subject: [PATCH] build: Source a CM-specific global BoardConfig

 * Read vendor/cm/config/BoardConfigCM.mk. Kind of surprised we haven't
   needed this until now.

Change-Id: I91cd89b14f6a5975e91f15dcfce1070502aa3861
---
 core/config.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/config.mk b/core/config.mk
index 4fd5bda..ab60701 100644
--- a/core/config.mk
+++ b/core/config.mk
@@ -187,6 +187,8 @@ FIND_LEAVES_EXCLUDES := $(addprefix --prune=, $(OUT_DIR) $(SCAN_EXCLUDE_DIRS) .r
 $(call project-set-path-variant,recovery,RECOVERY_VARIANT,bootable/recovery)
 
 -include vendor/extra/BoardConfigExtra.mk
+-include vendor/cm/config/BoardConfigCM.mk
+
 # The build system exposes several variables for where to find the kernel
 # headers:
 #   TARGET_DEVICE_KERNEL_HEADERS is automatically created for the current
